hibernate-testing/src/main/java/org/hibernate/testing/junit4/BaseNonConfigCoreFunctionalTestCase.java

```diff
@@ -483,9 +483,12 @@ protected boolean isCleanupTestDataRequired() {
 	protected void cleanupTestData() throws Exception {
 		Session s = openSession();
 		s.beginTransaction();
-		s.createQuery( "delete from java.lang.Object" ).executeUpdate();
-		s.getTransaction().commit();
-		s.close();
+		try {
```

Comment:
I think that doInHibernate automatically commits the transaction so the explicit commit is not required (?) but that's a good idea too, I will do that.